clients/include/complex.hpp

```diff
@@ -16,7 +16,11 @@ typedef struct hipsolverComplex
 #if __cplusplus >= 201103L
     hipsolverComplex() = default;
 #else
-    hipsolverComplex() {}
+    hipsolverComplex()
+        : x(0)
+        , y(0)
+    {
+    }
 #endif
```

Comment:
Could you do a clean build with `#error hipsolverComplex else branch` in that `#else`? I suspect `#if __cplusplus >= 201103L` is always true. If that's the case, we can just delete this branch of conditional logic entirely.